### DISCUSSION TITLE
Lesson 3: currency not being inserted in contract owner account.

### DISCUSSION THREAD

**Discussion by igormorgado** [smartcontractkit/full-blockchain-solidity-course-py#77] [Q&A] [answered]
Using the virtual machine environment, I have deployed the FundMe contract with the following code:

```
contract FundMe {
    mapping (address => uint256) public addressToAmountFunded;

    function fund() public payable {
        addressToAmountFunded[msg.sender] += msg.value;
    }
}
```

The code compiles and deploy nicely using lets say "ACCOUNT A".

Then I change to "ACCOUNT B", at same VM and proceed to a FUND() call. Filling the VALUE with a generous amount of currency lets say *987654 GWEY*. 

I can see the currency being debited from "ACCOUNT B", but nothing is added to "ACCOUNT A" (the account that has deployed the contract).

Is that correct? IS that expected? What I'm missing from the lesson?

**Comment by PatrickAlphaC** [comment, marked as answer]
Good question! This is correct.

You are not sends to fund a in this contract, you’re actually finding the contract itself.  So the contract will have the 987654 GWEI. 

To send the amount directly to another user, we use the transfer or send function. You’ll learn about those soon!

**Reply by igormorgado** [reply]
Thank you very much.
